addon/components/bs4/bs-collapse.js

```diff
@@ -1,2 +1,5 @@
-export { default } from 'ember-bootstrap/components/base/bs-collapse';
+import Collapse from 'ember-bootstrap/components/base/bs-collapse';
 
+export default Collapse.extend({
+  classNameBindings: ['collapse', 'in:show', 'collapsing']
```

Comment:
Sounds good. `classNameBindings` is problematic in a way, as it is a `mergedProperty` (http://emberjs.com/api/classes/Ember.Object.html#property_mergedProperties). So overriding does not work properly. So yeah, remove from base what is not needed in bs4, and add that to bs3. Make sense?